Source/Storages/Local/LocalKeyStorage.swift

```diff
@@ -39,33 +39,129 @@ import VirgilSDK
 
 internal class LocalKeyStorage {
     internal let identity: String
-    private let crypto: VirgilCrypto
+    internal let crypto: VirgilCrypto
+
+    private var keyPair: VirgilKeyPair?
```

Comment:
Should add possibility to reauthenticate user using biometrics after timer and/or after minimizing the app

Reply:
Need to make sure that key is not stored anywhere outside of this class for duration more than needed for one cryptographic operation

Reply:
>Should add possibility to reauthenticate user using biometrics after timer and/or after minimizing the app
Timer is bad for UX. Suddenly appeared check is weird. Will investigate possibility to observe minimizing app.